subsys/bluetooth/host/hci_core.c

```diff
@@ -4114,19 +4115,22 @@ static void rx_queue_put(struct net_buf *buf)
 
 static int bt_recv_unsafe(struct net_buf *buf)
 {
-	bt_monitor_send(bt_monitor_opcode(buf), buf->data, buf->len);
+	/* Don't pull the type, snice we still need it in the rx queue */
```

Comment:
Suggest to also add this comment to the `bt_send` function that does the same thing

Reply:
Sure, I'll add a comment

Reply:
Done